pol-core/pol/uoscrobj.cpp

```diff
@@ -4930,6 +4932,10 @@ BObjectImp* Client::set_script_member_id( const int id, int value )
   case MBR_DISABLE_INACTIVITY_TIMEOUT:
     disable_inactivity_timeout( value );
     return new BLong( disable_inactivity_timeout() );
+  case MBR_VISUAL_RANGE:
+    set_update_range( Clib::clamp_convert<u8>( value ) );
+    gd->script_defined_update_range = true;
```

Comment:
Would it be reasonable to allow setting to `uninit` to reset to whatever user had set before/default value and allow client to change it again?

Reply:
Resolving as this was addressed: setting to 0